libsolidity/ast/AST.cpp

```diff
@@ -38,6 +40,20 @@ using namespace std;
 using namespace solidity;
 using namespace solidity::frontend;
 
+namespace
+{
+TryCatchClause const*
+checkForClause(vector<ASTPointer<TryCatchClause>> const& clausesList, optional<string> stringPred = {})
```

Comment:
We use a leading `_` for function parameters
```suggestion
checkForClause(vector<ASTPointer<TryCatchClause>> const& _clausesList, optional<string> _stringPred = {})
```

Reply:
```suggestion
findClause(vector<ASTPointer<TryCatchClause>> const& _clauses, optional<string> _errorName = {})
```